features/step_definitions/user_steps.rb

```diff
@@ -139,8 +139,8 @@
 end
 
 Given /^I am logged out$/ do
-  visit destroy_user_session_path
-  visit destroy_admin_session_path
+  http_delete destroy_user_session_path
+  http_delete destroy_admin_session_path
 end
 
 Given /^I log out$/ do
```

Comment:
Does this step need to be updated to also click the confirm log out button, if the test is not using JS?

Reply:
It actually doesn't, this sends a DELETE request, which is what the confirm button does rather than the logout link. I'll be replacing this step with something that uses cookies anyway, as per your comment a bit lower.